examples/src/main/java/com/jsyn/examples/AudioPassThrough.java

```diff
@@ -51,7 +54,7 @@ private void test() {
 
         // We only need to start the LineOut. It will pull data from the LineIn.
         lineOut.start();
-        System.out.println("Audio passthrough started.");
+        LOGGER.debug("Audio passthrough started.");
```

Comment:
These example programs were meant to be run from the command line and print messages to the Terminal. System.out.println() is a standard way of doing that. 

Where will these logs appear?

Reply:
org.slf4j.Logger is using System.out as fallback.